planner/core/cache.go

```diff
@@ -115,7 +123,7 @@ func SetPstmtIDSchemaVersion(key kvcache.Key, stmtText string, schemaVersion int
 }
 
 // NewPlanCacheKey creates a new planCacheKey object.
-func NewPlanCacheKey(sessionVars *variable.SessionVars, stmtText, stmtDB string, schemaVersion int64) (kvcache.Key, error) {
+func NewPlanCacheKey(sessionVars *variable.SessionVars, stmtText, stmtDB string, schemaVersion int64, lastUpdatedSchemaVersion int64) (kvcache.Key, error) {
 	if stmtText == "" {
```

Comment:
Better to return an error if `lastUpdatedSchemaVersion` is zero for safety?

Reply:
Done.